blog.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Blog &mdash; Saejin Kwak Tanguay</title>
+    <link rel="stylesheet" href="./stylesheets/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="navbar">
+          <a href="mailto:saejinkwak@gmail.com">Contact</a>
+          <a href="./blog.html">Blog</a>
+          <a href="./portfolio.html">Portfolio</a>
+          <a href="./courses.html">Courses & Research</a>
+          <a href="./about.html">About</a>
+          <a href="./index.html">Home</a>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <h2>Blog</h2>
+      <article class = "blog-post">
```

Comment:
In general sectioning tags like articles should have some sort of heading.  It helps screen readers.